apps/web/pages/settings/license-key/new/index.tsx

```diff
@@ -0,0 +1,33 @@
+"use client";
+
+import LicenseRequired from "@calcom/features/ee/common/components/LicenseRequired";
+import { CreateANewLicenseKeyForm } from "@calcom/features/ee/deployment/licensekey/CreateLicenseKeyForm";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { WizardLayout, Meta } from "@calcom/ui";
+
+import { getServerSideProps } from "@lib/settings/license-keys/new/getServerSideProps";
+
+import PageWrapper from "@components/PageWrapper";
+
+const CreateNewOrganizationPage = () => {
```

Comment:
Nit: Should we rename this to `CreateNewLicenseKeyPage`?

Reply:
Wow good spot